### PR TITLE
Epauwels patch 1

### PR DESCRIPTION
**Pull request by epauwels**
Added end dates for all Tier1/Tier2 clusters at Ghent that were decommissioned

### CODE REVIEW

File: resource_files/UGent/resource_specs.json [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+  {
+    "resource": "dodrio_gpu_rome_a100_40",
+    "processors": 960,
+    "nodes": 20,
+    "ppn": 48,
+    "start_date": "2023-05-20"
+  },
+  {
+    "resource": "dodrio_gpu_rome_a100_80",
+    "processors": 960,
```
Comment (wpoely86) [inline resource_files/UGent/resource_specs.json:198]:
```suggestion
    "processors": 1920,
```

File: resource_files/UGent/resource_specs.json [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+  {
+    "resource": "dodrio_cpu_rome_512",
+    "processors": 5376,
+    "nodes": 42,
+    "ppn": 128,
+    "start_date": "2020-10-27"
+  },
+  {
+    "resource": "dodrio_gpu_rome_a100",
+    "processors": 960,
```
Comment (wpoely86) [inline resource_files/UGent/resource_specs.json:177]:
```suggestion
    "processors": 1920,
```

File: resource_files/UGent/resource_specs.json [hunk 3]
```diff
@@ -0,0 +1,210 @@
+[
+  {
+    "resource": "skitty",
+    "processors": 2592,
+    "nodes": 72,
+    "ppn": 36
```
Comment (WouterVanAssche) [inline resource_files/UGent/resource_specs.json:6]: missing comma at end of line

File: resource_files/UGent/resource_specs.json [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+    "ppn": 8,
+    "start_date": "2010-01-01"
+    "end_date": "2015-10-30"
+  },
+  {
+    "resource": "gulpin",
+    "processors": 1088,
+    "nodes": 34,
+    "ppn": 32,
+    "start_date": "2011-06-10"
```
Comment (WouterVanAssche) [inline resource_files/UGent/resource_specs.json:91]: missing comma at end of line

File: resource_files/UGent/resource_specs.json [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+    "end_date": "2015-11-23"
+  },
+  {
+    "resource": "dugtrio",
+    "processors": 192,
+    "nodes": 16,
+    "ppn": 12,
+    "start_date": "2011-09-05"
```
Comment (WouterVanAssche) [inline resource_files/UGent/resource_specs.json:99]: missing comma at end of line

File: resource_files/UGent/resource_specs.json [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+    "end_date": "2015-11-23"
+  },
+  {
+    "resource": "raichu",
+    "processors": 1024,
+    "nodes": 64,
+    "ppn": 16,
+    "start_date": "2012-06-15"
```
Comment (WouterVanAssche) [inline resource_files/UGent/resource_specs.json:107]: missing comma at end of line

File: resource_files/UGent/resource_specs.json [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+    "end_date": "2018-01-15"
+  },
+  {
+    "resource": "muk",
+    "processors": 8448,
+    "nodes": 528,
+    "ppn": 16,
+    "start_date": "2011-02-14"
```
Comment (WouterVanAssche) [inline resource_files/UGent/resource_specs.json:115]: missing comma at end of line

File: resource_files/UGent/resource_specs.json [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+    "nodes": 128,
+    "ppn": 96,
+    "start_date": "2020-08-04"
+  },
+  {
+    "resource": "slaking",
+    "processors": 1728,
+    "nodes": 12,
+    "ppn": 144,
+    "start_date": "2020-12-09"
```
Comment (WouterVanAssche) [inline resource_files/UGent/resource_specs.json:130]: missing comma at end of line

**Review by WouterVanAssche — CHANGES_REQUESTED** [review]
There are a couple lines with missing commas. I've marked them with comments.
The rest seems Ok.

File: resource_files/UGent/resource_specs.json (hunk 8 above)
Comment (epauwels) [reply resource_files/UGent/resource_specs.json]: All commas should now be in place

File: resource_files/UGent/resource_specs.json [hunk 9]
```diff
@@ -0,0 +1,210 @@
...
+    "nodes": 96,
+    "ppn": 36,
+    "start_date": "2018-01-24"
+  },
+  {
+    "resource": "delcatty",
+    "processors": 2560,
+    "nodes": 160,
+    "ppn": 16,
+    "start_date": "2013-05-15"
```
[the remark this answers is not in the document]
Comment (epauwels) [reply resource_files/UGent/resource_specs.json]: fixed

File: resource_files/UGent/resource_specs.json [hunk 10]
```diff
@@ -0,0 +1,210 @@
...
+    "ppn": 24,
+    "start_date": "2014-12-11"
+    "end_date": "2021-03-18"
+  },
+  {
+    "resource": "golett",
+    "processors": 4800,
+    "nodes": 200,
+    "ppn": 24,
+    "start_date": "2015-01-13"
```
[the remark this answers is not in the document]
Comment (epauwels) [reply resource_files/UGent/resource_specs.json]: fixed

File: resource_files/UGent/resource_specs.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (epauwels) [reply resource_files/UGent/resource_specs.json]: fixed

File: resource_files/UGent/resource_specs.json [hunk 11]
```diff
@@ -0,0 +1,210 @@
...
+    "nodes": 10,
+    "ppn": 32,
+    "start_date": "2019-08-09"
+  },
+  {
+    "resource": "kirlia",
+    "processors": 576,
+    "nodes": 16,
+    "ppn": 36,
+    "start_date": "2020-02-13"
```
[the remark this answers is not in the document]
Comment (epauwels) [reply resource_files/UGent/resource_specs.json]: fixed

File: resource_files/UGent/resource_specs.json [hunk 12]
```diff
@@ -0,0 +1,210 @@
...
+    "end_date": "2023-05-22"
+  },
+  {
+    "resource": "gengar",
+    "processors": 1552,
+    "nodes": 194,
+    "ppn": 8,
+    "start_date": "2008-11-01"
```
[the remark this answers is not in the document]
Comment (epauwels) [reply resource_files/UGent/resource_specs.json]: fixed

File: resource_files/UGent/resource_specs.json [hunk 13]
```diff
@@ -0,0 +1,210 @@
...
+    "end_date": "2014-06-10"
+  },
+  {
+    "resource": "gastly",
+    "processors": 448,
+    "nodes": 56,
+    "ppn": 8,
+    "start_date": "2009-12-01"
```
[the remark this answers is not in the document]
Comment (epauwels) [reply resource_files/UGent/resource_specs.json]: fixed

File: resource_files/UGent/resource_specs.json [hunk 14]
```diff
@@ -0,0 +1,210 @@
...
+    "end_date": "2015-11-23"
+  },
+  {
+    "resource": "haunter",
+    "processors": 1344,
+    "nodes": 168,
+    "ppn": 8,
+    "start_date": "2010-01-01"
```
[the remark this answers is not in the document]
Comment (epauwels) [reply resource_files/UGent/resource_specs.json]: fixed

File: resource_files/UGent/resource_specs.json (hunk 4 above)
Comment (epauwels) [reply resource_files/UGent/resource_specs.json]: fixed

File: resource_files/UGent/resource_specs.json (hunk 5 above)
Comment (epauwels) [reply resource_files/UGent/resource_specs.json]: fixed

File: resource_files/UGent/resource_specs.json (hunk 6 above)
Comment (epauwels) [reply resource_files/UGent/resource_specs.json]: fixed

File: resource_files/UGent/resource_specs.json (hunk 7 above)
Comment (epauwels) [reply resource_files/UGent/resource_specs.json]: fixed

**Review by WouterVanAssche — APPROVED** [review]

**Pull request merged**